packages/sw-cli/src/cli/index.js

```diff
@@ -370,14 +388,164 @@ class SWCli {
     });
   }
 
-  /**
-   * This method will generate the file manifest only.
-   * @return {Promise} The promise returned here will be used to exit the
-   * node process cleanly or not.
-   */
-  buildFileManifest() {
-    // TODO: Build File Manifest
-    return Promise.resolve();
+  _saveConfigFile() {
+    return inquirer.prompt([
+      {
+        name: 'saveConfig',
+        message: 'Last Question - Would you like to save these settings to ' +
+          'a config file?',
+        type: 'confirm',
+        default: true,
+      },
+    ])
+    .then((results) => {
+      return results.saveConfig;
+    })
+    .catch((err) => {
+      logHelper.error(
+        errors['unable-to-get-save-config'],
+        err
+      );
+      throw err;
+    });
+  }
+
+  _buildFileManifestFromGlobs(manifestFilePath, globs) {
+    let globbedFiles = [];
+    globs.forEach((globPattern) => {
```

Comment:
I sometimes hesitate to use `reduce()`, since they can lead to inscrutable code, but I'm just throwing this out there because this is the ideal use case:

```js
const globbedFiles = globs.reduce((accumulated, globPattern) => {
  const files = this._getFileManifestDetails(globPattern);
  return accumulated.concat(files);
}, []);
```

Or, this might work too, though I'm not 100% sure it will flatten the arrays in the way you want it:

```js
const globbedFiles = [...globs.map((globPattern) => this._getFileManifestDetails(globPattern))];
```

Or, you know, keep your `forEach()` if you'd prefer.